emsdk.py

```diff
@@ -1360,7 +1383,7 @@ def build_binaryen_tool(tool):
     return False
 
   # Make
-  success = make_build(build_root, build_type, 'x64' if tool.bitness == 64 else 'Win32')
+  success = make_build(build_root, build_type)
```

Comment:
I guess maybe the removal of this third unused argument could be its own NFC (non-functional-change)?  But its not a big deal if you want to leave it as part of this one..

Reply:
Are you sure this is unused?

Reply:
You decide, I had those changes written before VS 2022 support was added upstream so when I was updating my old changes including VS 2022 support to the new upstream state, this change was some kind of leftover of the code adding the VS 2022 support.

Reply:
I always prefer smaller and separate changes.  So I would prefer a separate PR in this case if you have time.